src/Filters/Filters.php

```diff
@@ -43,4 +45,20 @@ public function getIterator(): iterable
     {
         yield from $this->filters;
     }
+
+    public function getPreparedValues(ApplicatorInterface $applicator): array
+    {
+        return $this->filters
```

Comment:
Well yes, it looks quite easier .. I must admit it .. 
Few points still.. `Map<string, any>` strictly validates that `key` type is a `string`.
And I don't prefer merging arrays by `+` operator, since it hides that is `merging` not `addition`.